docs/readmes/dp/debug_logs.md

```diff
@@ -7,8 +7,171 @@ id: debug_logs title: View Logs hide_title: true
+2022-04-25 15:41:22,772 DEBUG Sending TR069 request to set CPE parameter values: {'SAS Radio Enable': True}
+2022-04-25 15:41:22,773 DEBUG State transition from <SetParameterValuesState> to <wait_set_params>
+2022-04-25 15:41:22,773 DEBUG Sending TR069 message: SetParameterValues {'ParameterList': {'ParameterValueStruct': [{'Name': 'Device.DeviceInfo.SAS.RadioEnable', 'Value': {'type': 'xsd:boolean', 'Data': '1'}}], 'arrayType': 'cwmp:ParameterValueStruct[1]'}}
+2022-04-25 15:41:22,777 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 723
+2022-04-25 15:41:22,837 DEBUG Handling TR069 message: SetParameterValuesResponse {'Status': '0'}
+2022-04-25 15:41:22,838 INFO Successfully configured CPE parameters!
+2022-04-25 15:41:22,839 DEBUG State transition from <WaitSetParameterValuesState> to <check_get_params>
+2022-04-25 15:41:22,840 DEBUG State transition from <GetParametersState> to <check_wait_get_params>
+2022-04-25 15:41:22,840 DEBUG Sending TR069 message: GetParameterValues {...}
+2022-04-25 15:41:22,849 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 3242
+2022-04-25 15:41:22,916 DEBUG Handling TR069 message: GetParameterValuesResponse {...}
+```
+
+d) Notify Domain Proxy state and TR069 session end
+
+```console
+2022-04-25 15:41:22,917 DEBUG State transition from <WaitGetParametersState> to <end_session>
+2022-04-25 15:41:22,917 DEBUG State transition from <BaicellsQRTBEndSessionState> to <notify_dp>
+2022-04-25 15:41:23,046 DEBUG Updating desired config based on sas grant
+2022-04-25 15:41:23,046 DEBUG Sending TR069 message: DummyInput {}
+```
+
+`notify_dp` state is the state where `enodebd` requests grant state from Domain Proxy for given eNB (based on eNB serial).
+As of writing this documentation, received data is stored in `enodebd` desired device configuration
+and will be applied to eNB on next TR069 session.
+
+## AGW/enodebd <-> Domain Proxy
+
+### AGW
+
+To debug AGW communication with Domain Proxy, look for `control_proxy` logs on the AGW calling gRPC API of Domain Proxy:
+
+```console
+journalctl -xu magma@control_proxy -r
+
+Apr 21 11:58:00 FreedomFi-Gateway control_proxy[1220289]: 2022-04-21T11:58:00.002Z [127.0.0.1 -> dp_service-controller.codistaging.dp.freedomfi.com,8443] "POST /DPService/GetCBSDState
+Apr 21 11:57:55 FreedomFi-Gateway control_proxy[1220289]: 2022-04-21T11:57:54.991Z [127.0.0.1 -> dp_service-controller.codistaging.dp.freedomfi.com,8443] "POST /DPService/GetCBSDState
+Apr 21 11:57:50 FreedomFi-Gateway control_proxy[1220289]: 2022-04-21T11:57:50.044Z [127.0.0.1 -> dp_service-controller.codistaging.dp.freedomfi.com,8443] "POST /DPService/GetCBSDState
+Apr 21 11:57:45 FreedomFi-Gateway control_proxy[1220289]: 2022-04-21T11:57:45.002Z [127.0.0.1 -> dp_service-controller.codistaging.dp.freedomfi.com,8443] "POST /DPService/GetCBSDState
+```
+
+This is gRPC so it's binary coded, and there isn't much information from tcpdump capture on this end of communication.
```

Comment:
What does `This` refer to? It is better to be explicit and "call the devil :smiling_imp: by its name".

Reply:
Rephrased.